src/controller/src/clusters.rs

```diff
@@ -83,12 +83,27 @@ pub struct ReplicaAllocation {
     /// The number of credits per hour that the replica consumes.
     #[serde(deserialize_with = "mz_repr::adt::numeric::str_serde::deserialize")]
     pub credits_per_hour: Numeric,
+    /// Whether instances of this type can be created
+    #[serde(default = "return_true")]
+    pub enabled: bool,
```

Comment:
I don't understand what this is needed for. Maybe the doc comment can be expanded to describe its higher level purpose as well.

Reply:
(Changed to `disabled`) This allows us to specify a `free` replica size that cannot actually be created.